lib/internal/readline/interface.js

```diff
@@ -75,8 +75,15 @@ const { StringDecoder } = require('string_decoder');
 const kHistorySize = 30;
 const kMaxUndoRedoStackSize = 2048;
 const kMincrlfDelay = 100;
-// \r\n, \n, or \r followed by something other than \n
-const lineEnding = /\r?\n|\r(?!\n)/g;
+/**
+ * The end of a line is signaled by either one of the following:
+ *  - \r\n
+ *  - \n
+ *  - \r followed by something other than \n
+ *  - \u2028 (Unicode 'LINE SEPARATOR')
+ *  - \u2029 (Unicode 'PARAGRAPH SEPARATOR')
+ */
+const lineEnding = /\r?\n|\r(?!\n)|\u2028|\u2029/g;
```

Comment:
would it make more sense to just use `/[\s\S]/`, which will catch whatever the JS engine considers a newline?

Reply:
I am not sure I follow your suggestion :confused: 

I think that `/[\s\S]/` would capture just about anything:
![Screenshot at 2025-04-02 15-03-36](https://github.com/user-attachments/assets/b444bba1-94c8-4fc5-800e-0a52df2009dd)

since `\s` captures white-space characters and `\S` captures non white-space ones ([mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_expressions/Character_classes))

Reply:
oh sorry i meant `/[^\s\S]/` :-)

Reply:
But I also don't think that that would help :confused: 

![Screenshot at 2025-04-02 15-21-12](https://github.com/user-attachments/assets/653f76a0-1849-442f-aa7c-1c74d061575a)

Reply:
lol sorry, i'm traveling and can't think straight, it seems :-)

`[\s\S]` is for `.` *including* newlines, so i'll resolve this and it's fine as-is

Reply:
no worries, it happens to the best of us! thanks a lot for looking at the PR in any case :heart_hands: